### PR TITLE
Genereate types_swagger_doc_generated.go for expapi

### PR DESCRIPTION
**Pull request by caesarxuchao**
After #10933, we use comments rather than description tag to add description for fields in types.go.

This PR makes the related scripts cover expapi/, and removes the description tags in expapi/v1/types.go and expapi/types.go. Note that this PR only generates types_swagger_doc_generated.go for expapi, it doesn't expose the swagger spec of expapi to user.

This is also required by #13471 to pass the tests.

Hopefully this one can be merged quickly, otherwise we need to do many rebase as the expapi evolves quickly.

@nikhiljindal @andronat @bgrant0607 @lavalamp

### CODE REVIEW

File: pkg/expapi/types.go [hunk 1]
```diff
@@ -68,11 +69,16 @@ type ReplicationControllerDummy struct {
...
+	// Kind of the referent; More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#types-kinds"
+	Kind string `json:"kind,omitempty"`
+	// Namespace of the referent; More info: http://releases.k8s.io/HEAD/docs/user-guide/namespaces.md
+	Namespace string `json:"namespace,omitempty"`
+	// Name of the referent; More info: http://releases.k8s.io/HEAD/docs/user-guide/identifiers.md#names
+	Name string `json:"name,omitempty"`
+	// API version of the referent
+	APIVersion string `json:"apiVersion,omitempty"`
+	// Subresource name of the referent
+	Subresource string `json:"subresource,omitempty" decription:"subresource name of the referent"`
```
Comment (nikhiljindal) [inline pkg/expapi/types.go]: Remove decription. There is a typo which is why it wasnt caught by the test

File: pkg/expapi/v1/types.go [hunk 2]
```diff
@@ -118,39 +124,52 @@ type HorizontalPodAutoscalerList struct {
...
+	// Versions are versions for this third party object
+	Versions []APIVersion `json:"versions,omitempty"`
 }
 
 type ThirdPartyResourceList struct {
 	v1.TypeMeta `json:",inline"`
-	v1.ListMeta `json:"metadata,omitempty" description:"standard list metadata; see http://docs.k8s.io/api-conventions.md#metadata"`
 
-	Items []ThirdPartyResource `json:"items" description:"items is a list of schema objects"`
+	// Standard list metadata;
```
Comment (nikhiljindal) [inline pkg/expapi/v1/types.go]: Dont need the semicolon at end

File: hack/update-generated-swagger-docs.sh [hunk 3]
```diff
@@ -24,13 +24,13 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 kube::golang::setup_env
 
 function generate_version() {
-	local version=$1
+	local prefix=$1
```
Comment (nikhiljindal) [inline hack/update-generated-swagger-docs.sh]: better to call it groupVersion instead of prefix?

**Comment by nikhiljindal** [comment]
Mostly LGTM. A few nits

**Comment by caesarxuchao** [comment]
Comments addressed. PTAL. Thank you @nikhiljindal

**Comment by nikhiljindal** [comment]
LGTM, thanks for the fixes.
Feel free to add the LGMT tag, once you have squashed the commits.

**Comment by caesarxuchao** [comment]
Adding LGTM per above comment.

**Comment by caesarxuchao** [comment]
Shippable failure is due to #11942. Re-apply the LGTM.

**Comment by caesarxuchao** [comment]
@k8s-bot test this please

**Comment by caesarxuchao** [comment]
```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/service.go:278
Expected error:
    <*errors.errorString | 0xc2082a0260>: {
        s: "Only 0 pods started out of 3",
    }
    Only 0 pods started out of 3
not to have occurred
```

**Comment by caesarxuchao** [comment]
@k8s-bot test this please

**Comment by caesarxuchao** [comment]
```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/container_probe.go:74
pod never became ready
Expected error:
    <*errors.errorString | 0xc20802d280>: {
        s: "timed out waiting for the condition",
    }
    timed out waiting for the condition
not to have occurred
```

**Comment by caesarxuchao** [comment]
@ArtfulCoder, could you merge this one?

e2e is flaky, I pasted the last two e2e failures and they are different.

**Comment by caesarxuchao** [comment]
@k8s-bot test this please

**Comment by caesarxuchao** [comment]
@k8s-bot test this please

**Comment by caesarxuchao** [comment]
```/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubectl.go:322
Sep  2 15:52:59.853: Error running &{/jenkins-master-data/jobs/kubernetes-pull-build-test-e2e-gce/workspace/kubernetes/platforms/linux/amd64/kubectl [kubectl --server=https://130.211.156.145 --kubeconfig=/var/lib/jenkins/jobs/kubernetes-pull-build-test-e2e-gce/workspace/.kube/config describe node pull-e2e-7-minion-1zl9] []  <nil>  format mismatch: DecimalSI vs. BinarySI
 [] <nil> 0xc208283ea0 exit status 1 <nil> true [0xc2080526c8 0xc208052718 0xc208052768] [0xc2080526c8 0xc208052718 0xc208052768] [0xc2080526f8 0xc208052740] [0x685b70 0x685b70] 0xc208221740}:
Command stdout:

stderr:
format mismatch: DecimalSI vs. BinarySI
```

**Comment by caesarxuchao** [comment]
@k8s-bot test this please

**Comment by caesarxuchao** [comment]
```
Kubernetes e2e suite.Services should serve multiport endpoints from pods
```

**Pull request merged**